app/scripts/lib/rpc-method-middleware/handlers/switch-ethereum-chain.js

```diff
@@ -65,7 +65,12 @@ async function switchEthereumChainHandler(
 
   const { origin } = req;
 
-  const { chainId } = req.params[0];
+  const { chainId, uuid } = req.params[0];
```

Comment:
Can you describe how this changes the dapp API?

Reply:
so really the second param I've added here is for internal use (not part of the public API). This is used by the `addEthereumChain` handler [here](https://github.com/MetaMask/metamask-extension/pull/17421/files#diff-b3485a1ea9943fd380364f73a53295289f2ccd3817423d4f245b5102f8620b3cR170) and [here](https://github.com/MetaMask/metamask-extension/pull/17421/files#diff-b3485a1ea9943fd380364f73a53295289f2ccd3817423d4f245b5102f8620b3cR314) to pass a single value that can be used [to quickly switch network](https://github.com/MetaMask/metamask-extension/pull/17421/files#diff-93f17dab5026baf840aec766ac5b6ebcd35d5bb0abb45e2c3f41791423f2a488R71) without having [to again loop through all the networkConfigurations](https://github.com/MetaMask/metamask-extension/pull/17421/files#diff-93f17dab5026baf840aec766ac5b6ebcd35d5bb0abb45e2c3f41791423f2a488R29) and determine which network we want to prompt the user to switch to?